go/src/koding/remoteapi/models/j_group_role.go

```diff
@@ -16,7 +16,7 @@ type JGroupRole struct {
 	// id
 	ID string `json:"_id,omitempty"`
 
-	// is configurable
+	// is configureable
```

Comment:
I'll remove it in a separate PR. For now I fixed it manually.

What would be better instead of spellcheck linter is some bot that would send PR with fixes automagically. 😎